src/entities/user.ts

```diff
@@ -23,22 +24,27 @@ export class User {
   @Column({ default: 'admin' })
   role: string;
 
-  @Column()
+  @Column({ name: 'created_at' })
   createdAt: Date;
 
-  @Column({ default: false })
+  @Column({ default: false, name: 'is_active' })
   isActive: boolean;
 
   @Column({
     type: 'enum',
```

Comment:
Better keep it in type varchar or char, because later you could have other purpose types